tests/lib/snaps/test-snapd-python-webserver/meta/snap.yaml

```diff
@@ -0,0 +1,18 @@
+name: test-snapd-python-webserver
+version: 16.04-3
+summary: Python based example webserver
+description: |
+ Show random XKCD comic via a build-in webserver
+ This is meant as a fun example for a snappy package.
+apps:
+ test-snapd-python-webserver:
+  command: ./bin/test-snapd-python-webserver
```

Comment:
Can we please use 4-space indents on these files as well? A single space is too close for quickly skimming through.